utils/build-script-impl

```diff
@@ -2383,41 +2490,16 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
             exit 1
         fi
 
-        if [ "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARGETS}" ] ; then
-            # If cross compiling tools, install into a deployment target specific subdirectory.
-            if [[ ! "${SKIP_MERGE_LIPO_CROSS_COMPILE_TOOLS}" ]] ; then
-                target_install_destdir="${BUILD_DIR}"/intermediate-install/"${deployment_target}"
-            else
-                target_install_destdir="${INSTALL_DESTDIR}"/"${deployment_target}"
-            fi
-        else
-            target_install_destdir="${INSTALL_DESTDIR}"
-        fi
-
         echo "--- Installing ${product} ---"
-        build_dir=$(build_directory ${deployment_target} ${product})
+        build_dir=$(build_directory ${host} ${product})
 
         set -x
 
-        DESTDIR="${target_install_destdir}" "${CMAKE}" --build "${build_dir}" -- ${INSTALL_TARGETS}
+        DESTDIR="${INSTALL_DESTDIR}" "${CMAKE}" --build "${build_dir}" -- ${INSTALL_TARGETS}
         { set +x; } 2>/dev/null
     done
 
-done
-
-if [[ "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARGETS}" ]] && [[ ! "${SKIP_MERGE_LIPO_CROSS_COMPILE_TOOLS}" ]] ; then
-    echo "--- Merging and running lipo for ${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARGETS[@]} ---"
```

Comment:
The purpose of this step is to lipo together the cross-compiled compilers, and copy over the standard library, so that we would get a consistent installation layout ready to be deployed onto a target.  This step is Darwin-specific, but other targets will need something similar.